.github/meson-android-aarch64.ini

```diff
@@ -0,0 +1,18 @@
+[binaries]
+c       = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android28-clang'
```

Comment:
why assume those paths? this looks like very wrong to me because:

* each distro/OS have the ndk installed in different path
* each ndk use to drop the bins in different places
* ndk provides a script that builds the environment to use "gcc, make, ranlib,..." without having to use those aliases

Reply:
can you let the ndk prefix path be in a separate variable?

Reply:
1) this files are useful only for github, so they don't need to be good for all distros.
2+3) the script we are using in sys/android-build has been deprecated for a while and they suggest to use these paths instead. Again, we don't need these paths to be good for user usecases, just for github ci.